scripts/publishTemporarilyUnavailablePageImage.ps1

```diff
@@ -0,0 +1,3 @@
+# Rough script for quick build-and-release to a particular env
+docker build -t ntbscontainerregistry.azurecr.io/temporarily-unavailable ./temporarily-unavailable/
+docker push ntbscontainerregistry.azurecr.io/temporarily-unavailable
```

Comment:
Have you run this script?

Reply:
No, where do I need to run it

Reply:
It looks like you would run it from the root directory of the project. You'll need to install docker first though! I've already got docker installed, so if you'd prefer you can hand this task over to me.

Reply:
Having tried it I think the file needs updating, and it looks like I need some sort of authentication to push the image (I can build it fine). Maybe it would be best to hand this over to you, thanks

Reply:
Great, I'll take a look at this today